src/8890_patches/picks.sh

```diff
@@ -5,118 +5,112 @@ set -eE
 
 ## Project: LineageOS/android_frameworks_base
 
-./vendor/lineage/build/tools/repopick.py 320714 --path ./frameworks/base # SystemUI: add burnIn protection
```

Comment:
The paths will need to be added back in order for this script to not fail. Also, some picks cause errors and must be commented out. Check existing file.

Reply:
#!/bin/bash

# Abort early on error
set -eE

## Project: LineageOS/android_frameworks_base

./vendor/lineage/build/tools/repopick.py 320714 --path ./frameworks/base # SystemUI: add burnIn protection
./vendor/lineage/build/tools/repopick.py 320765 --path ./frameworks/base # SystemUI: Fix shutter sound
./vendor/lineage/build/tools/repopick.py 321337 --path ./frameworks/base # Deprioritize important developer notifications
./vendor/lineage/build/tools/repopick.py 321338 --path ./frameworks/base # Allow disabling important developer notifications
./vendor/lineage/build/tools/repopick.py 321339 --path ./frameworks/base # Allow disabling USB notifications
./vendor/lineage/build/tools/repopick.py 322825 --path ./frameworks/base # Add CHANNEL_MODE_DUAL_CHANNEL constant
./vendor/lineage/build/tools/repopick.py 322826 --path ./frameworks/base # Add Dual Channel into Bluetooth Audio Channel Mode developer options menu
./vendor/lineage/build/tools/repopick.py 322827 --path ./frameworks/base # Allow SBC as HD audio codec in Bluetooth device configuration
./vendor/lineage/build/tools/repopick.py 322828 --path ./frameworks/base # Explicitly make SBC Dual Channel an optional (HD) codec
./vendor/lineage/build/tools/repopick.py 326692 --path ./frameworks/base # Skip screen on animation when wake and unlock via biometrics
./vendor/lineage/build/tools/repopick.py 327687 --path ./frameworks/base # biometrics: Allow to disable haptic feedback on success authentication

## Project: LineageOS/android_lineage-sdk

./vendor/lineage/build/tools/repopick.py 327684 --path ./lineage-sdk # lineage: res: Introduce config to disable biometrics success haptic feedback
./vendor/lineage/build/tools/repopick.py 327685 --path ./lineage-sdk # lineage-sdk: Add preference keys for biometrics success haptic feedback

## Project: LineageOS/android_packages_apps_Backgrounds

./vendor/lineage/build/tools/repopick.py 328434 --path ./packages/apps/Backgrounds # Revert "Include a limited set of wallpapers on devices with small /system"
./vendor/lineage/build/tools/repopick.py 328435 --path ./packages/apps/Backgrounds # Remove jpeg check from generate_smallvariants.py
./vendor/lineage/build/tools/repopick.py 328436 --path ./packages/apps/Backgrounds # [WIP] New wallpapers

## Project: LineageOS/android_packages_apps_Bluetooth

./vendor/lineage/build/tools/repopick.py 322838 --path ./packages/apps/Bluetooth # SBC Dual Channel (SBC HD Audio) support

## Project: LineageOS/android_packages_apps_CarrierConfig

./vendor/lineage/build/tools/repopick.py 320749 --path ./packages/apps/CarrierConfig # Fix voicemail for Bouygues Telecom France

## Project: LineageOS/android_packages_apps_DeskClock

./vendor/lineage/build/tools/repopick.py 324512 --path ./packages/apps/DeskClock # DeskClock: Remove unused drawables
./vendor/lineage/build/tools/repopick.py 325330 --path ./packages/apps/DeskClock # DeskClock: Add FLAG_IMMUTABLE
./vendor/lineage/build/tools/repopick.py 325331 --path ./packages/apps/DeskClock # DeskClock: Make sure int values are unique
./vendor/lineage/build/tools/repopick.py 325332 --path ./packages/apps/DeskClock # DeskClock: Material Me? No YOU!
./vendor/lineage/build/tools/repopick.py 325367 --path ./packages/apps/DeskClock # DeskClock: Add gradle
./vendor/lineage/build/tools/repopick.py 325953 --path ./packages/apps/DeskClock # DeskClock: Fix left button display
./vendor/lineage/build/tools/repopick.py 326383 --path ./packages/apps/DeskClock # DeskClock: Remove legacy support
./vendor/lineage/build/tools/repopick.py 326994 --path ./packages/apps/DeskClock # DeskClock: Remove compatibility to old APIs

## Project: LineageOS/android_packages_apps_Gallery2

./vendor/lineage/build/tools/repopick.py 328558 --path ./packages/apps/Gallery2 # Gallery3D: remove hw keys check from TimeLineSlotView

## Project: LineageOS/android_packages_apps_LineageParts

./vendor/lineage/build/tools/repopick.py 326739 --path ./packages/apps/LineageParts # Trust: Add option to always disable USB gadgets

## Project: LineageOS/android_packages_apps_Messaging

./vendor/lineage/build/tools/repopick.py 324405 --path ./packages/apps/LineageParts # Add support for text/vCard and text/vCalendar MIME type

## Project: LineageOS/android_packages_apps_Settings

./vendor/lineage/build/tools/repopick.py 322833 --path ./packages/apps/Settings # Add Dual Channel into Bluetooth Audio Channel Mode developer options menu
./vendor/lineage/build/tools/repopick.py 327686 --path ./packages/apps/Settings # Settings: Add biometrics success haptic feedback preference

## Project: LineageOS/android_packages_apps_SetupWizard

./vendor/lineage/build/tools/repopick.py 323145 --path ./packages/apps/SetupWizard # SUW: Enable status bar outside of FinishActivity

## Project: LineageOS/android_packages_apps_Trebuchet

./vendor/lineage/build/tools/repopick.py 317967 --path ./packages/apps/Trebuchet # Enable crossfade when changing theme
./vendor/lineage/build/tools/repopick.py 317969 --path ./packages/apps/Trebuchet # Expose themed icon setting in ThemePicker
./vendor/lineage/build/tools/repopick.py 326714 --path ./packages/apps/Trebuchet # NavbarButtonsViewController: Support navbar layout inversion

## Project: LineageOS/android_packages_inputmethods_LatinIME

./vendor/lineage/build/tools/repopick.py 325405 --path ./packages/inputmethods/LatinIME # LatinIME: fit more emoji in a page
./vendor/lineage/build/tools/repopick.py 325748 --path ./packages/inputmethods/LatinIME # LatinIME: add attr for vertical hint padding
# ./vendor/lineage/build/tools/repopick.py 325406 --path ./packages/inputmethods/LatinIME # LatinIME: Initial You
./vendor/lineage/build/tools/repopick.py 325437 --path ./packages/inputmethods/LatinIME # LatinIME: remove unused drawables
./vendor/lineage/build/tools/repopick.py 325438 --path ./packages/inputmethods/LatinIME # LatinIME: show emoji key when language switch key is not showing
# ./vendor/lineage/build/tools/repopick.py 328160 --path ./packages/inputmethods/LatinIME # LatinIME: update follow system theme label
./vendor/lineage/build/tools/repopick.py 328419 --path ./packages/inputmethods/LatinIME # LatinIME: Update emojis

## Project: LineageOS/android_packages_modules_Permission

./vendor/lineage/build/tools/repopick.py 317972 --path ./packages/modules/Permission # PermissionController: Enable usage timeline for all permission groups
./vendor/lineage/build/tools/repopick.py 317973 --path ./packages/modules/Permission # PermissionController: Show up to 7 days of permission usage history

## Project: LineageOS/android_packages_modules_adb

./vendor/lineage/build/tools/repopick.py 326385 --path ./packages/modules/adb # adb: Bring back support for legacy FunctionFS

## Project: LineageOS/android_packages_overlays_Lineage

./vendor/lineage/build/tools/repopick.py 326989 --path ./packages/overlays/Lineage # NavigationBarNoHint: Set taskbar_frame_height to 0dp

## Project: LineageOS/android_packages_services_Telecomm

./vendor/lineage/build/tools/repopick.py 327458 --path ./packages/services/Telecomm # Telecomm: Squashed phone_type switch support

## Project: LineageOS/android_system_bt

./vendor/lineage/build/tools/repopick.py 319781 --path ./system/bt # Add option to change eSCO Transport Unit Size to 16
./vendor/lineage/build/tools/repopick.py 322834 --path ./system/bt # Increase maximum Bluetooth SBC codec bitrate for SBC HD
./vendor/lineage/build/tools/repopick.py 322835 --path ./system/bt # Explicit SBC Dual Channel (SBC HD) support
./vendor/lineage/build/tools/repopick.py 322836 --path ./system/bt # Allow using alternative (higher) SBC HD bitrates with a property

## Project: LineageOS/android_system_core

./vendor/lineage/build/tools/repopick.py 317912 --path ./system/core # init: Don't run update_sys_usb_config if /data isn't mounted

## Project: LineageOS/android_vendor_lineage

# ./vendor/lineage/build/tools/repopick.py 317981 --path ./vendor/lineage # device_config: Save discrete app op history for more permissions
# ./vendor/lineage/build/tools/repopick.py 317982 --path ./vendor/lineage # device_config: Keep up to 7 days of permission usage history
./vendor/lineage/build/tools/repopick.py 325012 --path ./vendor/lineage # lineage: Opt-in to shipping full recovery image by default
./vendor/lineage/build/tools/repopick.py 327902 --path ./vendor/lineage # overlays: Use rounded corners in SystemUI
./vendor/lineage/build/tools/repopick.py 328170 --path ./vendor/lineage # config: Disable remote keyguard animation until it's fixed
./vendor/lineage/build/tools/repopick.py 328453 --path ./vendor/lineage # Update default wallpaper